website/src/categories/components/combobox/usage.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Combobox usage
+tags: usage
+eleventyNavigation:
+  parent: Combobox
+  key: ComboboxUsage
+---
+
+<section class="no-heading">
+
+<div class="ds-example">
+
+<sl-combobox multiple value='["0","1"]'>
```

Comment:
Is it ok that in the example, when we open the page, the combobox is already focused and opened?

<img width="937" alt="image" src="https://github.com/user-attachments/assets/f0cf517e-a510-47ea-971a-3bb06f15d0b7" />